<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>400</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>LCD Image Converter</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QTabWidget" name="tabWidget">
      <property name="currentIndex">
       <number>-1</number>
      </property>
      <property name="tabsClosable">
       <bool>true</bool>
      </property>
      <property name="movable">
       <bool>true</bool>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>600</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuHello">
    <property name="title">
     <string>File</string>
    </property>
    <widget class="QMenu" name="menuRecent">
     <property name="title">
      <string>Recent</string>
     </property>
    </widget>
    <addaction name="actionNew_Image"/>
    <addaction name="actionNew_Font"/>
    <addaction name="actionOpen"/>
    <addaction name="menuRecent"/>
    <addaction name="actionRename"/>
    <addaction name="actionSave"/>
    <addaction name="actionSave_As"/>
    <addaction name="actionClose"/>
    <addaction name="actionClose_All"/>
    <addaction name="separator"/>
    <addaction name="actionConvert"/>
    <addaction name="actionConvert_All"/>
    <addaction name="separator"/>
    <addaction name="actionQuit"/>
   </widget>
   <widget class="QMenu" name="menuImage">
    <property name="title">
     <string>Image</string>
    </property>
    <widget class="QMenu" name="menuRotate">
     <property name="title">
      <string>Rotate</string>
     </property>
     <addaction name="actionImageRotate_90_Clockwise"/>
     <addaction name="actionImageRotate_180"/>
     <addaction name="actionImageRotate_90_Counter_Clockwise"/>
    </widget>
    <widget class="QMenu" name="menuShift">
     <property name="title">
      <string>Shift</string>
     </property>
     <addaction name="actionImageShiftLeft"/>
     <addaction name="actionImageShiftRight"/>
     <addaction name="actionImageShiftUp"/>
     <addaction name="actionImageShiftDown"/>
    </widget>
    <addaction name="actionImageFlip_Horizontal"/>
    <addaction name="actionImageFlip_Vertical"/>
    <addaction name="separator"/>
    <addaction name="menuShift"/>
    <addaction name="menuRotate"/>
    <addaction name="actionImageResize"/>
    <addaction name="actionImageAlign"/>
    <addaction name="actionImageInverse"/>
    <addaction name="actionImageTo_Grayscale"/>
    <addaction name="separator"/>
    <addaction name="actionImageImport"/>
    <addaction name="actionImageExport"/>
    <addaction name="actionEdit_in_external_tool"/>
   </widget>
   <widget class="QMenu" name="menuFont">
    <property name="title">
     <string>Font</string>
    </property>
    <addaction name="actionFontChange"/>
    <addaction name="separator"/>
    <addaction name="actionFontResize"/>
    <addaction name="actionFontAlign"/>
    <addaction name="actionFontInverse"/>
    <addaction name="actionFontPreview"/>
    <addaction name="actionFontToImage"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="actionHomePage"/>
    <addaction name="actionUpdates"/>
    <addaction name="actionAboutQt"/>
    <addaction name="actionAboutApp"/>
   </widget>
   <widget class="QMenu" name="menuOptions">
    <property name="title">
     <string>Options</string>
    </property>
    <widget class="QMenu" name="menuLanguage">
     <property name="title">
      <string>Language</string>
     </property>
     <addaction name="actionLanguageDefault"/>
    </widget>
    <addaction name="actionConversionOptions"/>
    <addaction name="actionExternal_editor"/>
    <addaction name="menuLanguage"/>
   </widget>
   <widget class="QMenu" name="menuEdit">
    <property name="title">
     <string>Edit</string>
    </property>
    <addaction name="actionEditUndo"/>
    <addaction name="actionEditRedo"/>
    <addaction name="separator"/>
    <addaction name="actionEditCopy"/>
    <addaction name="actionEditPaste"/>
   </widget>
   <addaction name="menuHello"/>
   <addaction name="menuEdit"/>
   <addaction name="menuImage"/>
   <addaction name="menuFont"/>
   <addaction name="menuOptions"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionNew_Image">
   <property name="text">
    <string>New Image...</string>
   </property>
  </action>
  <action name="actionNew_Font">
   <property name="text">
    <string>New Font...</string>
   </property>
  </action>
  <action name="actionOpen">
   <property name="text">
    <string>Open...</string>
   </property>
   <property name="shortcut">
    <string notr="true">Ctrl+O</string>
   </property>
  </action>
  <action name="actionSave">
   <property name="text">
    <string>Save</string>
   </property>
   <property name="shortcut">
    <string notr="true">Ctrl+S</string>
   </property>
  </action>
  <action name="actionSave_As">
   <property name="text">
    <string>Save As...</string>
   </property>
   <property name="shortcut">
    <string notr="true"/>
   </property>
  </action>
  <action name="actionClose">
   <property name="text">
    <string>Close</string>
   </property>
   <property name="shortcut">
    <string notr="true">Ctrl+W</string>
   </property>
  </action>
  <action name="actionQuit">
   <property name="text">
    <string>Quit</string>
   </property>
   <property name="shortcut">
    <string notr="true">Ctrl+Q</string>
   </property>
  </action>
  <action name="actionImageFlip_Horizontal">
   <property name="text">
    <string>Flip Horizontal</string>
   </property>
  </action>
  <action name="actionImageFlip_Vertical">
   <property name="text">
    <string>Flip Vertical</string>
   </property>
  </action>
  <action name="actionImageRotate_90_Clockwise">
   <property name="text">
    <string>90 Clockwise</string>
   </property>
   <property name="shortcut">
    <string notr="true">Ctrl+Right</string>
   </property>
  </action>
  <action name="actionImageRotate_180">
   <property name="text">
    <string>180</string>
   </property>
  </action>
  <action name="actionImageRotate_90_Counter_Clockwise">
   <property name="text">
    <string>90 Counter-Clockwise</string>
   </property>
   <property name="shortcut">
    <string notr="true">Ctrl+Left</string>
   </property>
  </action>
  <action name="actionImageInverse">
   <property name="text">
    <string>Inverse</string>
   </property>
   <property name="shortcut">
    <string notr="true">Ctrl+I</string>
   </property>
  </action>
  <action name="actionConvert">
   <property name="text">
    <string>Convert...</string>
   </property>
   <property name="shortcut">
    <string notr="true">Ctrl+P</string>
   </property>
  </action>
  <action name="actionSetupConversion">
   <property name="text">
    <string>Conversion...</string>
   </property>
  </action>
  <action name="actionImageImport">
   <property name="text">
    <string>Import...</string>
   </property>
  </action>
  <action name="actionImageExport">
   <property name="text">
    <string>Export...</string>
   </property>
  </action>
  <action name="actionImageResize">
   <property name="text">
    <string>Resize...</string>
   </property>
  </action>
  <action name="actionFontChange">
   <property name="text">
    <string>Change...</string>
   </property>
  </action>
  <action name="actionFontResize">
   <property name="text">
    <string>Resize...</string>
   </property>
  </action>
  <action name="actionFontInverse">
   <property name="text">
    <string>Inverse</string>
   </property>
  </action>
  <action name="actionAboutApp">
   <property name="text">
    <string>About...</string>
   </property>
   <property name="shortcut">
    <string notr="true">F1</string>
   </property>
  </action>
  <action name="actionConversionOptions">
   <property name="text">
    <string>Conversion...</string>
   </property>
  </action>
  <action name="actionLanguageDefault">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>English</string>
   </property>
  </action>
  <action name="actionRename">
   <property name="text">
    <string>Rename...</string>
   </property>
  </action>
  <action name="actionFontPreview">
   <property name="text">
    <string>Preview...</string>
   </property>
  </action>
  <action name="actionHomePage">
   <property name="text">
    <string>Home Page...</string>
   </property>
  </action>
  <action name="actionFontToImage">
   <property name="text">
    <string>To Image</string>
   </property>
  </action>
  <action name="actionConvert_All">
   <property name="text">
    <string>Convert All</string>
   </property>
  </action>
  <action name="actionEdit_in_external_tool">
   <property name="text">
    <string>Edit in external tool...</string>
   </property>
   <property name="shortcut">
    <string notr="true">Ctrl+E</string>
   </property>
   <property name="autoRepeat">
    <bool>true</bool>
   </property>
  </action>
  <action name="actionExternal_editor">
   <property name="text">
    <string>External editor...</string>
   </property>
  </action>
  <action name="actionImageShiftLeft">
   <property name="text">
    <string>Left</string>
   </property>
   <property name="shortcut">
    <string notr="true">Ctrl+Shift+Left</string>
   </property>
  </action>
  <action name="actionImageShiftRight">
   <property name="text">
    <string>Right</string>
   </property>
   <property name="shortcut">
    <string notr="true">Ctrl+Shift+Right</string>
   </property>
  </action>
  <action name="actionImageShiftUp">
   <property name="text">
    <string>Up</string>
   </property>
   <property name="shortcut">
    <string notr="true">Ctrl+Shift+Up</string>
   </property>
  </action>
  <action name="actionImageShiftDown">
   <property name="text">
    <string>Down</string>
   </property>
   <property name="shortcut">
    <string notr="true">Ctrl+Shift+Down</string>
   </property>
  </action>
  <action name="actionEditUndo">
   <property name="text">
    <string>Undo</string>
   </property>
   <property name="shortcut">
    <string notr="true">Ctrl+Z</string>
   </property>
  </action>
  <action name="actionEditRedo">
   <property name="text">
    <string>Redo</string>
   </property>
   <property name="shortcut">
    <string notr="true">Ctrl+Y</string>
   </property>
  </action>
  <action name="actionUpdates">
   <property name="text">
    <string>Updates...</string>
   </property>
  </action>
  <action name="actionEditCopy">
   <property name="text">
    <string>Copy</string>
   </property>
   <property name="shortcut">
    <string notr="true">Ctrl+C</string>
   </property>
  </action>
  <action name="actionEditPaste">
   <property name="text">
    <string>Paste</string>
   </property>
   <property name="shortcut">
    <string notr="true">Ctrl+V</string>
   </property>
  </action>
  <action name="actionAboutQt">
   <property name="text">
    <string>About Qt...</string>
   </property>
  </action>
  <action name="actionImageTo_Grayscale">
   <property name="text">
    <string>To Grayscale</string>
   </property>
  </action>
  <action name="actionClose_All">
   <property name="text">
    <string>Close All</string>
   </property>
  </action>
  <action name="actionImageAlign">
   <property name="text">
    <string>Align...</string>
   </property>
  </action>
  <action name="actionFontAlign">
   <property name="text">
    <string>Align...</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
